truss/cli/cli.py

```diff
@@ -417,6 +444,44 @@ def deploy(
         )
 
 
+@chains.command(name="init")  # type: ignore
+@click.option(
+    "--target_directory",
+    type=str,
+    required=False,
+    help="Name of the chain to be deployed, if not given, the user will be prompted for a name",
+)
+@error_handling
+def chains_init(
+    target_directory: Optional[str],
+) -> None:
+    """
+    Initializes a chains project with hello.py
+    """
+    FILENAME = "main.py"
+    if not target_directory:
+        target_directory = input("Enter the target directory for the chain: ")
+    cur_path = os.getcwd()
+    abs_path = os.path.join(cur_path, target_directory)
+    filename = os.path.join(abs_path, FILENAME)
+    if os.path.exists(filename):
+        raise click.UsageError(
+            f"Cannot init chains project with {filename}. Path already exists"
+        )
+    user_friendly_path = os.path.join(target_directory, FILENAME)
+    rich.print(f"Creating {user_friendly_path}...\n")
+
+    os.mkdir(abs_path)
+    with open(filename, "w") as f:
+        f.write(HELLO_PY)
+
+    rich.print(
+        "Next steps:\n",
+        f"Run `python {user_friendly_path}` locally by adding `chains.run_local()` to your main.py file\n",
```

Comment:
Why not include a `run_local()` section in the example? It seems more user-friendly to have a "all batteries" included.